docs/getting-started/service-entities-roles-permissions.md

```diff
@@ -25,10 +24,6 @@ If you haven't set up your service yet, [create your first service](/first-servi
 1. On the _Overview_ page, click **Go to Entities**, or from the main menu (left sidebar) click the **Entities** icon.
    The _Entities_ page opens. Here you see all the entities in your service.
 
-:::tip
-When creating an Entity, make sure you do not use a reserved name for the name of the Entity or for any of its fields. Go [here](https://github.com/amplication/amplication/blob/master/packages/amplication-server/src/core/entity/reservedNames.ts) to check the list of reserved names.
-:::
-
```

Comment:
@dericksozo I think this tip is helpful in the docs even if the app has an in-app message. Can you please revert this change in the PR?